source/api/presentation/3.0/index.md

```diff
@@ -46,41 +46,38 @@ This is a work in progress and may change without any notices. Implementers shou
 
-The objective of the IIIF (pronounced "Triple-Eye-Eff") Presentation API is to provide the information necessary to allow a rich, online viewing environment for structured digital objects to be presented to a human user, often in conjunction with the [IIIF Image API][image-api]. This is the sole purpose of the API and therefore descriptive information is given in a way that is intended for humans to read, but not semantically available to machines. In particular, it explicitly does __not__ aim to provide metadata that would drive a search engine for discovering unknown objects.
+The objective of the IIIF (pronounced "Triple-Eye-Eff") Presentation API is to provide the information necessary to allow a rich, online viewing environment for structured digital objects to be presented to a human user, often in conjunction with the [IIIF Image API][image-api]. This is the sole purpose of the API and therefore descriptive information is given in a way that is intended for humans to read, but not semantically available to machines. In particular, it explicitly does __not__ aim to provide metadata that would allow a search engine to index digital objects.
 
 Implementations of this specification will be able to:
 
-  * display to the user digitized images, video, audio and other content types associated with a particular physical or born-digital object ;
-  * allow the user to navigate between multiple views of the object, either sequentially or hierarchically ;
-  * display descriptive information about the object, view or navigation structure to provide context to the user ;
+  * display to the user digitized images, video, audio, and other content types associated with a particular physical or born-digital object;
+  * allow the user to navigate between multiple views or time extents of the object, either sequentially or hierarchically;
+  * display descriptive information about the object, view or navigation structure to provide context to the user;
   * and provide a shared environment in which both publishers and users can annotate the object and its content with additional information.
 
 The following are __not__ in scope:
 
-  * The discovery or selection of interesting objects is not directly supported; however properties to reference further resources are available.
-  * Search within the object; which is described by the [IIIF Content Search API][search-api].
+  * Provision of metadata for harvesting and discovery is not directly supported. Properties to reference further descriptive resources are available, and their use is encouraged.
+  * Search within the object, which is described by the [IIIF Content Search API][search-api].
 
 Note that in the following descriptions, "object" is used to refer to the object that has been digitized or a born-digital compound object, and "resources" refer to the digital resources and content that are the result of that digitization or digital creation process.
 
-###  1.2. Motivating Use Cases
```

Comment:
There should be a reference to the cookbook, with the motivation that it provides many worked examples.